docker/travis_test/Dockerfile

```diff
@@ -58,7 +58,7 @@ RUN zypper in -y -C \
        'perl(Cpanel::JSON::XS)' \
        'perl(Crypt::DES)' \
        'perl(Cwd)' \
-       'perl(DBD::Pg)' \
+       'perl(DBD::Pg) >= 3.7.4' \
```

Comment:
this will cause a minor conflict with https://github.com/os-autoinst/openQA/pull/2308 . I prefer to have my PR in first as then we do not even need to change this line here anymore but I can handle the conflict resolution on my side as well :)

Reply:
Looks like i was too slow. 😃

Reply:
when you rebase just skip this change, the zypper line should just install openQA-devel